code/game/gamemodes/vampire/vampire_chan.dm

```diff
@@ -0,0 +1,46 @@
+/datum/game_mode/vampire/changeling
+	name = "vampire_changeling"
+	config_tag = "vampchan"
+	restricted_jobs = list("AI", "Cyborg")
+	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Blueshield", "Nanotrasen Representative", "Magistrate", "Chaplain", "Internal Affairs Agent", "Nanotrasen Navy Officer", "Special Operations Officer", "Syndicate Officer", "Solar Federation General")
+	protected_species = list("Machine")
+	required_players = 1 //Testing, TODO blah balh
```

Comment:
You know the drill

Reply:
This is exactly why I write these comments, thanks for catching lol